app/views/maps/view_map.html.erb

```diff
@@ -0,0 +1,104 @@
+<%= stylesheet_link_tag "https://api.mapbox.com/mapbox.js/plugins/leaflet-fullscreen/v1.0.1/leaflet.fullscreen.css" %>
+
+<%= javascript_include_tag 'maps' %>
+<%= javascript_include_tag '//api.tiles.mapbox.com/mapbox.js/plugins/leaflet-omnivore/v0.3.1/leaflet-omnivore.min.js' %>
+<%= javascript_include_tag "https://maps.googleapis.com/maps/api/js?key=AIzaSyAxlBXzYwdeaOMKZgx_UNBp2qBtdD0L_9g" %>
+<%= javascript_include_tag 'https://unpkg.com/leaflet.gridlayer.googlemutant@0.7.0/Leaflet.GoogleMutant.js' %>
+<%= javascript_include_tag '/lib/image-sequencer/dist/image-sequencer.js' %>
+
+<div class= "row" id="knitter-map-pane" style="height: 450px; width: 100%;  margin-bottom: 1%; z-index: 5; float: none; max-width: none;"> </div>
+<button type="button" class="btn btn-outline-info btn-sm" onClick="$('.share-link').toggle()">Embed code</button>
+<div class="share-link" style="display:none;">
+    <%= render :partial => 'maps/share' %>
+</div>
+<a href="/maps/<%= @map.slug %>">
+    <button class=  "btn btn-primary" style= "position: absolute; top: 380px;left: 10px; padding: 10px; z-index: 40;" >
+        <i class="fas fa-pencil-alt"> </i> Edit this map
+    </button>
+</a>
+<div class="container-fluid">
+  <div class="text-center lead-text">
+  <p class="text-muted"> a Community Atlas of </p>
+  <h2><%= @map.location %> </h2>
+  <p> <%= link_to "@#{@map.author}", "/profile/#{@map.author}" unless @map.author == 'anonymous' %> </p>
+  </div>
+  <div class="row text-center">
+      <div class="col-md-4 offset-md-4 text-block">
+          <img class="mx-auto d-block" style="max-height: 75px; max-width: 75px;" src="<%= asset_path('balloon1.png') %>">
+          <h3>Make your own map </h3>
+          <p>Use a kite, balloon, pole or drone to take an aerial photo and tell your own visual story of the place </p>
+          <br> <br>
+          <p>
+          <a class="btn btn-primary btn-lg" href= 'https://store.publiclab.org/collections/mapping-kits'>
+              Get a mapping kit<i class="fa fa-angle-double-right fa-fw" style="font-size:18px;color:white;"></i>
+          </a>
+          <a class="btn btn-primary btn-lg" href="<%= new_map_url %>">
+              Add a new map <i class="fa fa-plus fa-fw" style="font-size:12px;color:white;"></i>
+          </a>
+          </p>
+
+      </div>
+  </div>
+</div>
+  <br><br>
+<div class="all-maps light-blue">
+    <br>
+    <h3 class="text-center"> Nearby Maps </h3>
+         <hr style="max-width: 500px;">
+    <br>
+    <%= render :partial => 'front_ui/maps', :locals => { :maps => @maps } %>
+</div>
+  <br><br>
+
+
+  <div class="container-fluid">
+      <div class="row text-center lead-text" id="info">
+          <div class="col-md-3 col-sm-5 offset-2">
+              <img class="mx-auto d-block" style="max-height: 75px; max-width: 75px;" src="<%= asset_path('quiz.png') %>">
+              <br>
+              <h3>Ask a question</h3>
+          </div>
+          <div class="col-md-3 col-sm-5 offset-2">
```

Comment:
I'm a little worried that the titles in one row and the texts in the next row may not fold properly when the window width is reduced -- can you check this in a mobile-width window to be sure? Thank you!

Reply:
oh yeah, this will look horrible in a mobile-view, will make the whole thing a row and find some text to add for uniformity